Spelling (#1610)
slackapi/bolt-js: docs/_future/request_time_off_tutorial.md

The function listener is declared in the file [`listeners/functions/request-approval.js`](https://github.com/slack-samples/bolt-js-request-time-off/blob/main/listeners/functions/request-approval.js).

> In the `request-approval.js` file, we are defining a handler function, `notifyApprover`, that our listener will run when the `ApprovalFunction` is invoked. `notifyApprover` takes in the provided parameters from the function and calls the API method [`client.chat.postMessage`](https://api.slack.com/methods/chat.postMessage) to send a [Block Kit](https://api.slack.com/block-kit) message with the submitted time off information to the manager. Additional buttons to approve or deny the request are also added into the message Block Kit&mdash;these will be handled by [Action](#action) handlers that are covered in the next section.

Author: Some people have very strong opinions on `into` and `in to`. This is by far one of my newest experiments in terms of semi-automated corrections. I'm happy to drop them or anything else.

Reviewer: I had to do a quick grammar refresh on the differences in usage between `in to` and `into` from [here](https://www.masterclass.com/articles/into-vs-in-to). Based on my learnings from that guide, I added in some suggestions for places where I feel like `in to` might make more sense versus `into` (such as when the "in" is part of a verb phrase), but for sure open to discussing this more if you'd like!

> In the `request-approval.js` file, we are defining a handler function, `notifyApprover`, that our listener will run when the `ApprovalFunction` is invoked. `notifyApprover` takes in the provided parameters from the function and calls the API method [`client.chat.postMessage`](https://api.slack.com/methods/chat.postMessage) to send a [Block Kit](https://api.slack.com/block-kit) message with the submitted time off information to the manager. Additional buttons to approve or deny the request are also added into the message Block Kit&mdash;these will be handled by [Action](#action) handlers that are covered in the next section.

Reviewer: ```suggestion
In the `request-approval.js` file, we are defining a handler function, `notifyApprover`, that our listener will run when the `ApprovalFunction` is invoked. `notifyApprover` takes in the provided parameters from the function and calls the API method [`client.chat.postMessage`](https://api.slack.com/methods/chat.postMessage) to send a [Block Kit](https://api.slack.com/block-kit) message with the submitted time off information to the manager. Additional buttons to approve or deny the request are also added in to the message Block Kit&mdash;these will be handled by [Action](#action) handlers that are covered in the next section.
```

> This `notifyApprover` handler function is then passed into a new `SlackFunction` registration so that it will be called when the `ApprovalFunction is invoked:

Reviewer: ```suggestion
This `notifyApprover` handler function is then passed in to a new `SlackFunction` registration so that it will be called when the `ApprovalFunction is invoked:
```

---
### Views and modals {#views-and-modals}

> Additional interactivity can be added into your next-generation Slack app through [Modals](/bolt-js/concepts#creating-modals) and [Views](/bolt-js/concepts#updating-pushing-views). Within the Request Time Off example, we can add an additional button to the time off request message a manager receives that opens a new modal.

Reviewer: ```suggestion
Additional interactivity can be added in to your next-generation Slack app through [Modals](/bolt-js/concepts#creating-modals) and [Views](/bolt-js/concepts#updating-pushing-views). Within the Request Time Off example, we can add an additional button to the time off request message a manager receives that opens a new modal.
```

Having additional interactivity such as a modal could create a pathway for more functionality in the Request Time Off flow, such as allowing a manager to send a message to the requester to get additional information or clarification before approving or denying the request officially.

> Let's dive in and see how we can add a new button that launches a simple modal with a corresponding view!

Author: This change seems reasonable and means that the basic rule "don't write `in to`" can be applied. But, if someone feels strongly for `dive in to see`, I'm happy to drop.

#### Add in view handler

> Now that the "Open Modal" button's action handler has been added into the `request-approval.js` file, we can add in a view handler that will trigger additional functionality if the modal is submitted.

Reviewer: ```suggestion
Now that the "Open Modal" button's action handler has been added in to the `request-approval.js` file, we can add in a view handler that will trigger additional functionality if the modal is submitted.
```
